src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -378,12 +378,39 @@ def get_parameter_of_bus(self, bus: str, parameter: Parameter):
             ParameterNotFound: Raised if parameter does not exist
         """
         # TODO: Change private QM API to public when implemented.
+
+        if "_config" not in dir(self):
+            self._config = self.settings.to_qua_config()
+
+        config_keys = self._config["elements"][bus]
+
         if parameter == Parameter.LO_FREQUENCY:
-            return self._qm._elements[bus].input.lo_frequency  # type: ignore[union-attr] # pylint: disable=protected-access
-        if parameter == Parameter.GAIN:
-            return self._qm._elements[bus].input.gain  # type: ignore[union-attr] # pylint: disable=protected-access
+            if "mixInputs" in config_keys:
+                return self._config["elements"][bus]["mixInputs"]["lo_frequency"]
```

Comment:
Okei, I see before we were using the default classes and attributes from QM.. and now with our own its more complicated, I guess this will work then! 👌

Reply:
In the future we could simplify the code with a class, because right now, we are repeating a lot the:
```self._config["elements"][bus][Parameter]```
 But I guess its acceptable now, since you are not the first to do it even 👍